web/src/components/AnalyticsDashboard.tsx

```diff
@@ -48,6 +49,7 @@ export const AnalyticsDashboard = () => {
   const enabled = useRecoilValue(simulatorEnabled);
   useSimulationMonitor(enabled);
   useSimulator(enabled);
+  trackAnalyticsEvent("analytics-dashboard-page");
```

Comment:
This could fire multiple times due to how react works. Much safer to either:
* add a custom hook that uses useEffect to guarantee one event emit per mount
* add page tracking logic to the router (I prefer this method since it's automatic and react router should have a nice way to hook into route changes)

Reply:
yes, the second approach is way better! 
```
analytics.track('rtdm-change-page', {
  pathname: '/overview'
});
```